app/models/application_form.rb

```diff
@@ -47,6 +47,8 @@ class ApplicationForm < ApplicationRecord
   BRITISH_OR_IRISH_NATIONALITIES = %w[GB IE].freeze
   MAXIMUM_NUMBER_OF_COURSE_CHOICES = 3
 
+  BEGINNING_OF_FREE_SCHOOL_MEALS = Date.parse('1/9/1964')
```

Comment:
we could save the computer a very small amount of work with `Date.new(1964, 9, 1)`